projects/vdk-plugins/vdk-lineage/requirements.txt

```diff
@@ -0,0 +1,11 @@
+openlineage-integration-common
+openlineage-python
+pytest
+pytest-httpserver
+sqllineage
+sqlparse
+vdk-core
```

Comment:
Isn't the requirements.txt file only for testing dependencies, given that the rest are already listed in the setup.py file? It seems reasonable to me to not have them in two places in case the deps change at some point in the future.

Reply:
IntelliJ recognizes requrements.txt and installs them automatically. But having a clear separation may be better so I removed them.